src/brogue/Items.c

```diff
@@ -862,7 +862,7 @@ void pickUpItemAt(pos loc) {
         theItem->flags |= ITEM_PLAYER_AVOIDS; // explore shouldn't try to pick it up more than once.
         itemName(theItem, buf2, false, true, NULL); // include article
         sprintf(buf, "Your pack is too full to pick up %s.", buf2);
-        message(buf, 0);
+        messageWithColor(buf, &itemMessageColor, 0);
```

Comment:
```suggestion
        messageWithColor(buf, &badMessageColor, 0);
```

Reply:
That's fine by me. It's an improvement over white, and I prefer it over yellow. I'll hold off on further discussion if this what we're landing on.

Reply:
Works for me.